src/containers/Tenant/Diagnostics/Tablets/Tablets.tsx

```diff
@@ -0,0 +1,171 @@
+import {ArrowsRotateRight} from '@gravity-ui/icons';
+import type {Column as DataTableColumn} from '@gravity-ui/react-data-table';
+import {Icon, Label, Text} from '@gravity-ui/uikit';
+import {skipToken} from '@reduxjs/toolkit/query';
+
+import {ButtonWithConfirmDialog} from '../../../../components/ButtonWithConfirmDialog/ButtonWithConfirmDialog';
+import {EntityStatus} from '../../../../components/EntityStatus/EntityStatus';
+import {ResponseError} from '../../../../components/Errors/ResponseError';
+import {InternalLink} from '../../../../components/InternalLink';
+import {Loader} from '../../../../components/Loader';
+import {ResizeableDataTable} from '../../../../components/ResizeableDataTable/ResizeableDataTable';
+import routes, {createHref} from '../../../../routes';
+import {selectTabletsWithFqdn, tabletsApi} from '../../../../store/reducers/tablets';
+import {ETabletState} from '../../../../types/api/tablet';
+import type {TTabletStateInfo} from '../../../../types/api/tablet';
+import type {TabletsApiRequestParams} from '../../../../types/store/tablets';
+import {cn} from '../../../../utils/cn';
+import {DEFAULT_TABLE_SETTINGS} from '../../../../utils/constants';
+import {calcUptime} from '../../../../utils/dataFormatters/dataFormatters';
+import {useTypedDispatch, useTypedSelector} from '../../../../utils/hooks';
+import {mapTabletStateToLabelTheme} from '../../../../utils/tablet';
+import {getDefaultNodePath} from '../../../Node/NodePages';
+
+import i18n from './i18n';
+
+const b = cn('tablets-table');
+
+const columns: DataTableColumn<TTabletStateInfo & {fqdn?: string}>[] = [
+    {
+        name: 'Type',
+        get header() {
+            return i18n('Type');
+        },
```

Comment:
Why you define header property this way and not just `header: i18n('Type')`?

Reply:
Theoretically, i18n should be called inside components/actions.